op-verifier/contracts/OPDisputeGameVerifier.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.17;
+
+import { IEVMVerifier } from "@ensdomains/evm-verifier/contracts/IEVMVerifier.sol";
+import { RLPReader } from "@eth-optimism/contracts-bedrock/src/libraries/rlp/RLPReader.sol";
+import { StateProof, EVMProofHelper } from "@ensdomains/evm-verifier/contracts/EVMProofHelper.sol";
+import { Types } from "@eth-optimism/contracts-bedrock/src/libraries/Types.sol";
+import { Hashing } from "@eth-optimism/contracts-bedrock/src/libraries/Hashing.sol";
+import "./interfaces/IDisputeGame.sol";
```

Comment:
Are these published anywhere under `@eth-optimism`?

Reply:
Published on https://www.npmjs.com/package/@eth-optimism/contracts-bedrock

![image](https://github.com/ensdomains/evmgateway/assets/4103490/71db5613-b4c0-46dd-ae6a-fa6485b6b988)

Will change to use @eth-optimism/contracts-bedrock

Reply:
I'm facing `Error HH411: The library src, imported from @eth-optimism/contracts-bedrock/src/dispute/interfaces/IDisputeGame.sol, is not installed. Try installing it using npm.` due to they have migrated to use foundry absolute import path.

I have asked at https://github.com/NomicFoundation/hardhat/discussions/5049 but no answer yet